### ISSUE TITLE
Galaxy on Fire Alliance 项目仍在继续！加油！

### ISSUE THREAD

**Issue by Aisishare** [ClementLevi/GoFA-Remake#2]
这个游戏真的很好19年停运后我一直在找相关信息，一直没有头绪，没想到在github找到了！

**Comment by ClementLevi** [comment]
在做的，只是极度缺少有经验的架构策划，现在项目进展缓慢（几周commit一点东西的样子）。具体进度可以先看readme，如果我传了的话……

**Comment by Aisishare** [comment]
如果能复刻开私服真的太棒了！多说几句，似乎没人知道这个项目，一个人的力量是有限的，让更多的人参与，互相帮助会更快！我不懂编程，但如果您需要帮助，我可以做一些力所能及的事（比如：宣传、测试、注资、玩法教程 www.aisishare.com/alliances-guide/ 等等等）！坚信Alliance有一天会像 ( 红警 www.ra2ol.com ) 一样，永不凋零持续运行！加油！加油！加油！

**Comment by Trimatix** [comment]
Great stuff Aisishare, what you can bring is really important 🎉

ClementLevi, did you receive my message on Twitter? I'm a member of the English Galaxy on Fire Community, which is full of GOF:A fans, and I'm sure that we can find some more developers to contribute to this project!

If nothing else, we have archived screenshots and information etc. from GOF:A, which could help.
(and one of the actual developers of GOF is in this community, who occasionally answers a question or two 😮)

**Comment by ClementLevi** [comment]
> Great stuff Aisishare, what you can bring is really important 🎉
> 
> ClementLevi, did you receive my message on Twitter? I'm a member of the English Galaxy on Fire Community, which is full of GOF:A fans, and I'm sure that we can find some more developers to contribute to this project!
> 
> If nothing else, we have archived screenshots and information etc. from GOF:A, which could help. (and one of the actual developers of GOF is in this community, who occasionally answers a question or two 😮)

Any type of help is tremendously encouraging! So far the Chinese Gofa communities I have contact with has accumulated some of the art resources, including the original installer for Android devices, but considering copyrights issues and other relevant legal problems, we are not sure yet whether should we use them in actual remake development. And this applies to the gameplay as well, only after permission can we start converting our present designs into actual codes. And to what degree should this Remake version be similar to the original game should be reviewed by the copyrights holder, at least I think so. The design ideas, however, should be innovative for us remakers and game fans. We do have actually designed some new features, which have been reflected in the data structures schemas, for new potential gameplay and avoid plagiarism. So the remake version could be extending the original one.. or maybe not. All depend on the community echo and legal negotiation in the future, which still need us to prepare much more abundant project design for review.
Our main focus at present will remain on handling accumulating required knowledges for development, as we're short of hand, especially those experienced architecure designers or coders. After that, the overall project design will be on the way. This could take a few more weeks to months. So... take it easy! We're still making progress for preparation (very very slowly ;) )
A community with original developers could accelerate our development progress for certain. Is that a Discord channel? I have joined one of the [server: The Kaamo Club](https://discord.com/channels/226528541368385536/702920192807665684), where indeed some of the developers are present. But if there is more, I would appreciate to know about.
As for the Twitter message you mentioned, unfortunately I haven't received any in my inbox [@ClementLevi2](https://twitter.com/ClementLevi2). Maybe it is blocked by some sort of settings I did not intented to make or know. Anyway check again or simply feel free to contact me here!
